tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -63,6 +63,120 @@ public function test_wp_enqueue_script() {
+		wp_add_inline_script( 'ms-isinsa-1', 'console.log("after one");', 'after', true );
+		wp_add_inline_script( 'ms-isinsa-1', 'console.log("after two");', 'after' );
+		$output    = get_echo( 'wp_print_scripts' );
+		$expected  = "<script type='text/javascript' src='http://example.org/ms-isinsa-1.js' id='ms-isinsa-1-js' defer></script>\n";
+		$expected .= "<script type='text/javascript' id='ms-isinsa-1-js-after'>\n";
+		$expected .= "console.log(\"after one\");\n";
+		$expected .= "</script>\n";
+		$expected .= "<script type='text/javascript' id='ms-isinsa-1-js-after' type='text/template'>\n";
+		$expected .= "console.log(\"after two\");\n";
+		$expected .= "</script>\n";
+		$this->assertSame( $expected, $output );
+	}
+
+	/**
+	 * Test standalone after inline script.
+	 *
+	 * @ticket 12009
+	 * @dataProvider data_standalone_after_inline_script
+	 */
+	public function test_standalone_after_inline_script( $expected, $output, $message ) {
+		$this->assertSame( $expected, $output, $message );
+	}
+
+	public function data_standalone_after_inline_script() {
+		$data = array();
+
+		// If the main script with defer strategy has a `after` inline script; the inline script is not affected.
+		wp_enqueue_script( 'ms-isa-1', 'http://example.org/ms-isa-1.js', array(), null, array( 'strategy' => 'defer' ) );
+		wp_add_inline_script( 'ms-isa-1', 'console.log("after one");', 'after', true );
+		$output    = get_echo( 'wp_print_scripts' );
+		$expected  = "<script type='text/javascript' src='http://example.org/ms-isa-1.js' id='ms-isa-1-js' defer></script>\n";
+		$expected .= "<script type='text/javascript' id='ms-isa-1-js-after'>\n";
+		$expected .= "console.log(\"after one\");\n";
+		$expected .= "</script>\n";
+		array_push( $data, array( $expected, $output, 'Expected no type attribute for inline script.' ) );
+
+		// If the main script with async strategy has a `after` inline script; the inline script is not affected.
+		wp_enqueue_script( 'ms-isa-2', 'http://example.org/ms-isa-2.js', array(), null, array( 'strategy' => 'defer' ) );
+		wp_add_inline_script( 'ms-isa-2', 'console.log("after one");', 'after', true );
+		$output    = get_echo( 'wp_print_scripts' );
+		$expected  = "<script type='text/javascript' src='http://example.org/ms-isa-2.js' id='ms-isa-2-js' defer></script>\n";
+		$expected .= "<script type='text/javascript' id='ms-isa-2-js-after'>\n";
+		$expected .= "console.log(\"after one\");\n";
+		$expected .= "</script>\n";
+		array_push( $data, array( $expected, $output, 'Expected no type attribute for inline script.' ) );
+
+		return $data;
+	}
+
+	/**
+	 * Test non standalone after inline script.
+	 *
+	 * @ticket 12009
+	 * @dataProvider data_non_standalone_after_inline_script
+	 */
+	public function test_non_standalone_after_inline_script( $expected, $output, $message ) {
+		$this->assertSame( $expected, $output, $message );
+	}
+
+	public function data_non_standalone_after_inline_script() {
```

Comment:
@kt-12 we need a docblock here.

Reply:
See prior comment around not being 100% certain we need docblocks here, but I think we should..

Reply:
This is added in PR #50. see comment-  https://github.com/10up/wordpress-develop/pull/49/files#r1154216339